activity_calendar/models.py

```diff
@@ -32,6 +32,17 @@
 def now_rounded():
     return timezone.now().replace(minute=0, second=0)
 
+class CoreActivityGrouping(models.Model):
+    """
+        A method to group certain core activities together. E.g. all boardgame evenings,
+        all roleplay-related actitvities, LARP-stuff, etc.
+    """
+    identifier = models.CharField(max_length=127, unique=True)
```

Comment:
It may be useful here to prevent empty strings here. Maybe add a `MinLengthValidator`? (https://docs.djangoproject.com/en/3.1/ref/validators/#minlengthvalidator)

Reply:
The field is required, so empty strings are already disallowed.